node/node.go

```diff
@@ -505,3 +506,9 @@ func (node *Node) initNodeConfiguration() (service.NodeConfig, chan p2p.Peer) {
 func (node *Node) AccountManager() *accounts.Manager {
 	return node.accountManager
 }
+
+// SetDNSFlag indicates whether use harmony dns server to get peer info for node syncing
+func (node *Node) SetDNSFlag(flag bool) {
+	utils.GetLogInstance().Info("SetDNSFlag", "flag", flag)
```

Comment:
This is a debug message, not an info as there is nothing more informative.

Reply:
yes, but it can be helpful to debug whether the command line is passed correctly